modules/serverless-build-events-kn.adoc

```diff
@@ -0,0 +1,76 @@
+[id="serverless-build-events-kn_{context}"]
+= Building events
+
+You can use the builder-like interface of the `kn event build` command to build an event. You can then send that event at a later time or use it in another context.
+
+.Procedure
+
+* Build an event:
++
+[source,terminal]
+----
+$ kn event build --field <field-name>=<value> --type <type-name> --id <id> --output <format>
+----
+where:
+** The `--field` option adds data to the event as a field-value pair. You can use it multiple times.
+** The `--type` option specifies the type of the event.
+** The `--id` option specifies the ID of the event.
+** You can use the `json` or `yaml` arguments with the `--output` option to change the output format of the event.
++
+All of these options are optional.
++
```

Comment:
As a suggestion, I'd say "flags" instead of "options" here so that the last line reads better, i.e. "all of these flags are optional". I think we refer to these as flags in other CLI docs.

Reply:
Might also be worth providing possible values, e.g. what "types" are available? How will a user know if they're not listed? Is there some help command we should include as well?

Reply:
@cardil Could you please provide the full list (or the most important items, if others are irrelevant) of arguments for the `--type` flag?

Reply:
Gah, I mis-looked. `--type` is a semi-arbitrary namespace-like string, the user can make it up themselves. Made it clear in the docs.